cdap-ui/app/cdap/components/BtnWithLoading/index.js

```diff
@@ -17,18 +17,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import LoadingSVG from 'components/LoadingSVG';
+import classnames from 'classnames';
 require('./BtnWithLoading.scss');
 
 export default function BtnWithLoading({
   className,
+  darker = false,
   label,
   loading,
   disabled,
   onClick
 }) {
   return (
     <button
-      className={`btn btn-with-loading ${className}`}
+      className={classnames(`btn btn-with-loading ${className}`,{
```

Comment:
nit: space after `,`

Reply:
i think you can simply do `classnames('btn btn-with-loading', className, { 'darker'.... })`